libs/eo/landing-page/shell/src/lib/eo-landing-page.integration.spec.ts

```diff
@@ -41,4 +48,49 @@ describe('EnergyOrigin landing page', () => {
 
     expect(await findEnergyOriginLogo()).toBeInTheDocument();
   });
+
+  it('displays the EnergiNet logo', async () => {
+    const { navigate } = await render(SpectacularAppComponent, {
+      imports: [
+        SpectacularFeatureTestingModule.withFeature({
+          featureModule: EoLandingPageShellModule,
+          featurePath: '',
+        }),
+        HttpClientTestingModule,
+      ],
+    });
+    await navigate('/');
+
+    expect(await findEnergiNetLogo()).toBeInTheDocument();
+  });
+
+  it('displays a telephone link', async () => {
+    const { navigate } = await render(SpectacularAppComponent, {
+      imports: [
+        SpectacularFeatureTestingModule.withFeature({
+          featureModule: EoLandingPageShellModule,
+          featurePath: '',
+        }),
+        HttpClientTestingModule,
+      ],
+    });
```

Comment:
suggestion: It looks like the test setup could be reused, for example using a `beforeEach` hook.